.docs/content/0.armonik/1.glossary.md

```diff
@@ -1,37 +1,79 @@
+## Node
 
-**Worker:** User-developed containerized software capable of performing one or several tasks depending on its implementation.  A worker can simply take input data and perform calculations on it to return a result. A worker can also submit new tasks that will be performed by different workers, other instances of itself or itself.
+In the context of Kubernetes, a node is a machine that runs containerized workloads as part of a Kubernetes cluster. A node can be a physical or virtual machine hosted in the cloud or on-premises.
+
+## Partition
+
+Logical segmentation of the Kubernetes cluster's pool of machines to distribute workloads according to usage. This feature is provided and handled by ArmoniK.
+
+## Payload
+
+Input data for a task that does not depend on any other task.
+
+## Pod
+
+Pods are the smallest deployable units of computing that one can create and manage in Kubernetes. A Pod is a group of one or more containers, with shared storage and network resources and a specification for how to run the containers. A Pod's contents are always co-located and co-scheduled and run in a shared context. A Pod models an application-specific "logical host": it contains one or more application containers which are relatively tightly coupled (see Kubernetes [documentation](https://kubernetes.io/docs/concepts/workloads/pods/)).
+
+## Polling agent
+
+Former term for scheduling agent.
+
+## Redis
+
+Redis is an open source (BSD licensed), in-memory data structure store used as a database, cache, message broker and streaming engine. Redis provides data structures such as strings, hashes, lists, sets and sorted sets with range queries, bitmaps, hyperloglogs, geospatial indexes and streams. Redis has built-in replication, Lua scripting, LRU eviction, transactions and different levels of on-disk persistence and provides high availability via Redis Sentinel and automatic partitioning with Redis Cluster (see Redis [documentation](https://redis.io/docs/about/)). In ArmoniK, Redis is used as a key-value cache for task data (such as payloads and results).
+
+## Scheduling agent
+
+Containerized software cohabiting with a worker within a pod, running a specific algorithm to determine which tasks "its" worker (the one with which it shares the pod) should perform. It also manages all interactions between the worker and the databases (retrieving/saving data, creating new tasks, etc.), as well as managing worker errors and retrying/resubmitting failed tasks when necessary. A scheduling agent, like a worker, exists within a single partition.
+
+## Session
+
+A session is a logical container for tasks and associated data (task statut, results, errors, etc). Every task is submitted within a session. An existing session can be resumed to retrieve data or submit new tasks. When a session is cancelled, all associated executions still in progress are interrupted.
```

Comment:
Is it really a container? I would start with " A session is a limited time when two systems communicate with each other. "

Reply:
In fact a session doesn't refer to a limited amount of time. So far session are eternal and never deleted. One can still retrieve data from an old session and submit new tasks.